examples/reporting/get_campaign_stats_to_csv.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+#
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This examples gets a basic campaign stats report for a given customer ID
+and writes them to a CSV file.
+
+The LoadFromStorage method is pulling credentials and properties from a
+"googleads.yaml" file. By default, it looks for this file in your home
+directory. For more information, see the "Caching authentication information"
+section of our README.
+
+Examples:
+    Write to file output.csv in the same directory as script with headers.
+        $ python get_campaign_stats_to_csv.py -c 0123456789 -o output.csv -w
+
+    Write to file output.csv in the same directory as script without headers.
+        $ python get_campaign_stats_to_csv.py -c 0123456789 -o output.csv -w
+"""
+
+import argparse
+import csv
+import sys
+
+from google.ads.google_ads.client import GoogleAdsClient
+from google.ads.google_ads.errors import GoogleAdsException
+
+
+def main(client, customer_id, output_file, write_headers):
+    """Demonstrates how to write rows returned from a Google Ads search_stream
+    request to a CSV file.
```

Comment:
Same as above, can you rephrase this comment onto a single line?

Reply:
Done.